lance/lance-common/src/main/java/org/apache/gravitino/lance/common/ops/gravitino/GravitinoLanceNamespaceWrapper.java

```diff
@@ -596,7 +598,7 @@ public CreateTableResponse createTable(
         break;
       case OVERWRITE:
         if (catalog.asTableCatalog().tableExists(tableIdentifier)) {
```

Comment:
if code reach here, `catalog.asTableCatalog().tableExists` will be called three times

Reply:
OK, I can call it once.  The only drawback is that the result of `tableExists` will be out-of-date.